src/tests/member.test.js

```diff
@@ -1,5 +1,85 @@
-// Basic test to avoid conflicts
-// REMOVE THIS TEST AND MAKE YOURS!
-test('should pass a basic test', () => {
-  expect(1 + 1).toBe(2);
+/* eslint-disable no-underscore-dangle */
+import request from 'supertest';
+import app from '../app';
+import Member from '../models/Member';
+import memberSeed from '../seeds/member';
+
+const mockedMemberId = memberSeed[0]._id;
+
+const invalidMemberMockUp = {
+  id: '64650c82fb9b0234915f4be6',
+};
+
+const modifiedMember = {
+  firstName: 'Juan',
+  lastName: 'Perez',
+  dni: 37254615,
+  birthday: '2005-08-12T03:00:00.000Z',
+  phone: '1676778390',
+  email: 'juanperez@yahoo.com',
+  city: 'Cordoba',
+  postalCode: '5000',
+  isActive: true,
+  membership: 'Classic',
+};
+
+beforeEach(async () => {
+  await Member.collection.insertMany(memberSeed);
+});
+
+afterEach(async () => {
+  await Member.collection.deleteMany();
+});
+
+describe('PUT /api/member', () => {
```

Comment:
You can consider including a scenario where invalid parameters are sent, resulting in a status code of 400. For example, passing the dni as a string format.
dni: "ewibiu6465"

Reply:
Solved!